File: 0137-single-number-ii/0137-single-number-ii.cs
public class Solution {
    public int SingleNumber(int[] nums) {
    int result=0;
    for(int i=0;i<nums.Length;i++){
        if(Array.IndexOf(nums,nums[i])==Array.LastIndexOf(nums,nums[i])) result=(nums[i]); 
    }
    return result;  
    }
}

File: 0137-single-number-ii/0137-single-number-ii.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var singleNumber = function(nums) {
    let result;
    for(let i=0;i<nums.length;i++){
        if(nums.indexOf(nums[i])==nums.lastIndexOf(nums[i])) result=(nums[i]); 
    }
    return result; 
};